### PR TITLE
22w46a client

### PR DESCRIPTION
**Pull request by apple502j**

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping [hunk 1]
```diff
@@ -8,12 +8,6 @@ CLASS net/minecraft/class_617 net/minecraft/client/render/entity/model/VexEntity
 	COMMENT   <th>Part Name</th><th>Parent</th><th>Corresponding Field</th>
 	COMMENT </tr>
 	COMMENT <tr>
-	COMMENT   <td>{@value EntityModelPartNames#HAT}</td><td>Root part</td><td>{@link #hat}</td>
-	COMMENT </tr>
-	COMMENT <tr>
-	COMMENT   <td>{@value EntityModelPartNames#HEAD}</td><td>Root part</td><td>{@link #head}</td>
```
Comment (YanisBft) [inline mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping:14]: Vex still has a head in its model

Comment (apple502j) [reply mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping:14]: Hmm? There is no field for it?

Comment (YanisBft) [reply mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping:14]: No field, but it's added to the ModelPartData in getTexturedModelData()

Comment (apple502j) [reply mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping:14]: Hmm, how do we document those?

Comment (YanisBft) [reply mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping:14]: In other model classes yeah, it seems the model parts added are documented, not the fields

**Review by YanisBft — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**